README.md

```diff
@@ -57,3 +57,137 @@ Contains implementations of an interface to launch jobs.  `cromwell.engine` will
 ### cromwell.engine
 
 Contains the Akka code and actor system to execute a workflow.  This layer should operate entirely on objects returned from the `cromwell.binding` layer.
+
+Web Server
+----------
+
+The `server` subcommand on the executable JAR will start an HTTP server which can accept WDL files to run as well as check status and output of existing workflows.
+
+### POST /workflows
+
+This endpoint accepts a POST request with a `multipart/form-data` encoded body.  The two elements in the body must be named `wdl` and `inputs`.  The `wdl` element contains the WDL file to run while the `inputs` contains a JSON file of the inputs to the workflow.
+
+```
+$ http --print=hbHB --form POST localhost:8000/workflows wdlSource=@src/main/resources/3step.wdl workflowInputs@inputs.json
+```
+
+Request:
+
+```
+POST /workflows HTTP/1.1
+Accept: */*
+Accept-Encoding: gzip, deflate
+Connection: keep-alive
+Content-Length: 730
+Content-Type: multipart/form-data; boundary=f1a9bd3079b14946bd1264efaa545c45
+Host: localhost:8000
+User-Agent: HTTPie/0.9.2
+
+--f1a9bd3079b14946bd1264efaa545c45
+Content-Disposition: form-data; name="wdlSource"
+
+task ps {
+  command {
+    ps
+  }
+  output {
+    File procs = "stdout"
+  }
+}
+
+task cgrep {
+  command {
+    grep '${pattern}' ${File in_file} | wc -l
+  }
+  output {
+    Int count = read_int("stdout")
+  }
+}
+
+task wc {
+  command {
+    cat ${File in_file} | wc -l
+  }
+  output {
+### POST /workflows
+    Int count = read_int("stdout")
+  }
+}
+
+workflow three_step {
+  call ps
+  call cgrep {
+    input: in_file=ps.procs
+  }
+  call wc {
+    input: in_file=ps.procs
+  }
+}
+
+--f1a9bd3079b14946bd1264efaa545c45
+Content-Disposition: form-data; name="workflowInputs"; filename="inputs.json"
+
+{
+    "three_step.cgrep.pattern": "..."
+}
+
+--f1a9bd3079b14946bd1264efaa545c45--
+```
+
+Response:
+
+```
+HTTP/1.1 201 Created
+Content-Length: 74
+Content-Type: application/json; charset=UTF-8
+Date: Tue, 02 Jun 2015 18:06:28 GMT
+Server: spray-can/1.3.3
+
+{
+    "id": "69d1d92f-3895-4a7b-880a-82535e9a096e",
+    "status": "WorkflowSubmitted"
```

Comment:
`WorkflowSubmitted` and `WorkflowSucceeded` feels [Smurfy](http://blog.codinghorror.com/new-programming-jargon/), how about just `Submitted` and `Succeeded`?